lib/restpack_serializer/serializable.rb

```diff
@@ -72,6 +73,24 @@ def add_custom_attributes(data)
       data.merge!(custom) if custom
     end
 
+    def apply_whitelist_and_blacklist(context)
+      if context[:attribute_blacklist].present?
+        self.class.serializable_attributes.each do |key, value|
+          if context[:attribute_blacklist].include? key
+            context[value[:include_method_name]] = false
+          end
+        end
+      end
+
+      if context[:attribute_whitelist].present?
+        self.class.serializable_attributes.each do |key, value|
+          unless context[:attribute_whitelist].include? key
+            context[value[:include_method_name]] = false
```

Comment:
Thanks, I'll raise an exception if both the whitelist and blacklist are present - only one should be allowed at a time